sysutils/puppet-agent/src/opnsense/service/conf/actions.d/actions_puppetagent.conf

```diff
@@ -0,0 +1,17 @@
+[start]
+command:/usr/local/etc/rc.d/puppet start
+parameters:
+type:script
+message:starting puppet agent
+
+[stop]
+command:/usr/local/etc/rc.d/puppet onestop
+parameters:
+type:script
+message:stop puppet agent
+
```

Comment:
Can you have a look at a similar plugin like Telegraf and copy `status`?

Reply:
I've added the status action.